src/platform/Tizen/ThreadStackManagerImpl.cpp

```diff
@@ -646,6 +646,12 @@ CHIP_ERROR ThreadStackManagerImpl::_RemoveInvalidSrpServices()
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR ThreadStackManagerImpl::_ClearAllSrpHostAndServices()
+{
+    // TODO : could not find info on Tizen thread apis
+    return CHIP_ERROR_NOT_IMPLEMENTED;
+}
```

Comment:
For Tizen just use the logic from `_RemoveInvalidSrpServices()` but without any "ifs":

```suggestion
CHIP_ERROR ThreadStackManagerImpl::_ClearAllSrpHostAndServices()
{
    for (auto it = mSrpClientServices.begin(); it != mSrpClientServices.end();)
    {
        ReturnErrorOnFailure(_RemoveSrpService(it->mInstanceName, it->mName));
        it = mSrpClientServices.erase(it);
    }
    return CHIP_NO_ERROR;
}
```

Reply:
ah great, thanks! applying